drag-and-drop.md

```diff
@@ -0,0 +1,126 @@
+# Drag-adn-Drop
+
+The proposal to provide native support Drag-and Drop opeerations:
+* drag and drop between widgets 
+* receive drag and drop URL(s) to application from a desktop or another application
```

Comment:
I don't think we should limit to URLs.
Perhaps this could be called Payload - it could be string, image, url all sorts right?

Reply:
Agree, the URL is too specific. A text string must be a general type for ALL platform.
Support of other Payload types can be added later.